src/cc_catalog_airflow/dags/provider_api_scripts/metropolitan_museum_of_art.py

```diff
@@ -0,0 +1,241 @@
+"""
+
+import argparse
+import os
+import common.requester as requester
+import common.storage.image as image
+import logging
+
+
+DELAY = 1.0  # time delay (in seconds)
+PROVIDER = 'met'
+
+logging.basicConfig(
+    format='%(asctime)s - %(name)s - %(levelname)s:  %(message)s',
+    level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+delayed_requester = requester.DelayedRequester(DELAY)
+image_store = image.ImageStore(provider=PROVIDER)
+
+
+def main(date=None):
+    """
+    This script pulls the data for a given date from the Metropolitan
+    Museum of Art API, and writes it into a .TSV file to be eventually
+    read into our DB.
+
+    Required Arguments:
+
+    date:  Date String in the form YYYY-MM-DD.  This is the date for
+           which running the script will pull data.
+    """
+
+    logger.info(f'Begin: Met Museum API requests for date: {date}')
+
+    fetch_the_object_id = _get_object_ids(date)
+    if fetch_the_object_id:
+        logger.info(f'Total object found {fetch_the_object_id[0]}')
+        _extract_the_data(fetch_the_object_id[1])
+
+    total_images = image_store.commit()
+    logger.info(f'Total CC0 images recieved {total_images}')
+
+
+def _get_object_ids(date):
+    query_params = ''
+    if date:
+        query_params = {
+            'metadataDate': date
+            }
+
+    endpoint = (
+        'https://collectionapi.metmuseum.org/public/collection/v1/objects'
+    )
+    response = _get_response_json(query_params, endpoint)
+
+    if response:
+        total_object_ids = response['total']
+        object_ids = response['objectIDs']
+    else:
+        logger.warning(f'No content available')
+        return None
+    return [total_object_ids, object_ids]
+
+
+def _get_response_json(
+        query_params,
+        endpoint,
+        retries=5,
+):
+    response_json = None
+
+    if retries < 0:
+        logger.error('No retries remaining.  Failure.')
+        raise Exception('Retries exceeded')
+
+    response = delayed_requester.get(
+        endpoint,
+        params=query_params,
+        timeout=60
+    )
+    if response is not None and response.status_code == 200:
+        try:
+            response_json = response.json()
+        except Exception as e:
+            logger.warning(f'Could not get response_json.\n{e}')
+            response_json = None
+
+    if response_json is None:
+        logger.warning(f'Bad response_json:  {response_json}')
+        logger.warning(
+            'Retrying:\n_get_response_json(\n'
+            f'    {endpoint},\n'
+            f'    {query_params},\n'
+            f'    retries={retries - 1}'
+            ')'
+        )
+        response_json = _get_response_json(
+            query_params,
+            endpoint=endpoint,
+            retries=retries - 1
+        )
+
+    return response_json
+
+
+def _extract_the_data(object_ids):
+    for i in object_ids:
+        _get_data_for_each_image(i)
+
+
+def _get_data_for_image(object_id):
```

Comment:
This function name was changed, but not the call to the function.  This means the script can't run in its current form.

However, when I changed the call to match the name of the function, the script works!